File: OdontoSysV2/OdontoSys/OdontoSysModel/src/main/java/com/odontosys/services/model/Tratamiento.java
package com.odontosys.services.model;


public class Tratamiento {
    
    private int idTratamiento;
    private String nombre;
    private String descripcion;
    private double costo;
    private Especialidad especialidad;

    public Tratamiento() {
        this.especialidad = new Especialidad();
    }

    public int getIdTratamiento() {
        return idTratamiento;
    }

    public void setIdTratamiento(int idTratamiento) {
        this.idTratamiento = idTratamiento;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    public double getCosto() {
        return costo;
    }

    public void setCosto(double costo) {
        this.costo = costo;
    }

    public Especialidad getEspecialidad() {
        return especialidad;
    }

    public void setEspecialidad(Especialidad especialidad) {
        this.especialidad = especialidad;
    }

    @Override
    public String toString() {
        return "Tratamiento{" + "idTratamiento=" + idTratamiento + ", nombre=" + nombre + ", descripcion=" + descripcion + ", costo=" + costo + ", especialidades=" + especialidad + '}';
    }
            
}

File: OdontoSysFrontEnd/OdontoSysModel/Services/Tratamiento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OdontoSysModel.Services
{
    public class Tratamiento
    {
        public int IdTratamiento { get; set; }
        public string Nombre { get; set; }
        public string Descripcion { get; set; }
        public double Costo { get; set; }
        public Especialidad Especialidad { get; set; }

        public Tratamiento()
        {
            this.Especialidad = new Especialidad();
        }

        public override string ToString()
        {
            return $"Tratamiento{{IdTratamiento={IdTratamiento}, Nombre={Nombre}, Descripcion={Descripcion}, Costo={Costo}, Especialidad={Especialidad}}}";
        }
    }
}
